starpy/manager.py

```diff
@@ -118,7 +119,10 @@ def registerEvent(self, event, function):
         """
         log.debug('Registering function %s to handle events of type %r',
                   function, event)
-        if isinstance(event, (str, unicode, type(None))):
+        types = [str, type(None)]
+        if sys.version_info <= (3, 0):
+            types.append(unicode)
```

Comment:
An alternative here would be to use the 'six' library and initialize types using six.string_types.

https://pythonhosted.org/six/

Reply:
I agree with not adding a new dependency for this one tiny feature.  Maybe I would have created a `def string_types(value):` inside manager.py with different implementation for old vs new to be called from both locations but that is not very important.

In either case I think this PR looks good enough and would prefer to see it merged.

Reply:
I'll could would add the one function for this case. Seems good catch to create a string_type method in manager.py. Let me know what your decide.